industry/products.md

```diff
@@ -17,11 +17,13 @@ They vary by:
 and other features, as well as underlying architecture.
 
 
-An advantage of using a machine translation provider is that the service is already working and can be integrated immediately. An alternative to using a machine translation provider is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+Some of the most popular MT providers offer off-the-shelf solutions, i.e. trained MT models that are ready-to-use and can be integrated immediately. An alternative is to train a model from scratch, deploy it and maintain it.
 
-Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+The main advantage of ready-to-use systems is that they are a one-size-fits-all solution, for example they can be used for any text domain. When used in an application, text segmentation and language identification are usually provided. On the other hand, a model trained from scratch might yield better performance on a specific domain if trained on a large amount of high-quality domain-specific bilingual data. However, training and maintaining such models can be a complex task and it requires specific competences.
+Some machine translation providers also offer large pre-trained models that can be fine-tuned. A common application scenario is one where a  pre-trained model is fine-tuned with a relatively small domain-specific data set to achieve high performance on specific contents.
```

Comment:
> Some machine translation providers also offer large pre-trained models that can be fine-tuned.

Pre-trained models are also increasingly available for free on the internet.

> A common application scenario is one where a pre-trained model is fine-tuned with a relatively small domain-specific data set to achieve high performance on specific contents.

This isn't really different from the customization in the cloud-based offerings like Google or Microsoft.

Reply:
Re-inserted the original wording above